### ISSUE TITLE
Fields for facility profile

### ISSUE THREAD

**Issue by dboyer** [open-apparel-registry/open-apparel-registry#634]
## Overview

These fields should be collected on the facility profile page

### Describe the solution you'd like

Facility details

- Official facility name (native language)
- Official facility name (English)
- Address
- Country
- Website
- Contact person
- Email
- Telephone number
- Description

Facility type
Dropdown menu:

- Cut and Sew / RMG
- Fabric Mill
- Dyehouse
- Laundry
- Finishing
- Tannery
- Spinning
- Ginning
- Other (Free text box)

Product type

Production type

Minimum order quantity

Average lead time

Number of workers

Male / female split [%]

Certifications/Standards/Regulations 

Membership bodies (awaiting list of options)

Head office information
(If different from facility address)

- Official office name
- Address
- Country
- Contact person
- Email
- Telephone number

### Additional Context

Some of this feedback on new fields was originally in #617. Moved here in order to keep issues organized.

**Comment by kellyi** [comment]
Does the "production type" field differ from "facility type"? If not, what set of options should be displayed in production type?

**Comment by jwalgran** [comment]
There is a discussion thread in the "Claim a Facility Text" Google Doc around `Official facility name` and an agreement that we should split this into two fields, with the `Official facility name` being in the native language and an additional field for the English language version. I have updated the issue description to reflect this.

**Comment by kellyi** [comment]
https://github.com/open-apparel-registry/open-apparel-registry/pull/650 adds the new fields discussed here with the exception of the following, for which I think we need more information:

- Product type
- Production type
- Certifications
- Membership bodies

I believe we're waiting on an enumerated/closed set of choices for "Certifications" and "Membership bodies".

If I remember correctly, we discussed making "Product type" into a field into which users could select from an existing set of tags or create new tags.

"Production type" may or may not differ from "Facility type", so we need to know if it does and, if so, what options it should have.

**Comment by kellyi** [comment]
I made discrete followup issues for handling

- product type #651 
- production type #652 
- certifications #653 
- membership bodies #654

**Comment by dboyer** [comment]
@kellyi Production type and Facility type are two separate fields. Facility type should include a drop-down menu with the options listed above. Production type should be a field where users can select from existing set of tags or create new tags. 

Product type should also be a field in which users select from an existing set of tags or create new tags.

There will be seed data for product type and production type. I'm waiting on that yet.

**Comment by dboyer** [comment]
Follow up on this. We need to redo the list of facility types to match the list below.

Cut and Sew / RMG
Dyehouse
Embellishments
Fabric Mill
Finishing
Ginning
Knitting
Laundry
Packing
Scouring
Screenprinting
Spinning
Stiching (Shoes)
Tannery
Warehouse
Weaving
Other (Free text box)

**Comment by kellyi** [comment]
> We need to redo the list of facility types to match the list below.

Sounds good. I'll push a new commit to #650 with that change.

**Comment by kellyi** [comment]
d67149dc1b112b4d868c3bbfff1a22a48a645f07 adjusts the facility type field as described above
